bigsi/cmds/variant_search.py

```diff
@@ -11,6 +11,11 @@ def flatten(l):
     return [item for sublist in l for item in sublist]
 
 
+def filter_bigsi_var_results(d):
+    d["results"] = [x for x in d.get("results", []) if x["genotype"] != "0/0"]
+    return d
```

Comment:
No, the example you mentioned is not a bug. If `d.get("results", [])` returns the default `[]`, then the statements inside the for loop will not be executed because there is no elements in the list. So there will be no error raised there.

Reply:
I don't know how the changes in this file get into my branch. This is not my code. I will revert this file.